app/models/user.rb

```diff
@@ -60,7 +60,7 @@ def enqueue_inactive_email
   end
 
   def able_to_edit_repo?(repo)
-    repo.user_name == github
+    repo.user_name == github.downcase
```

Comment:
It's okay. Thanks!

And by the way, this line of code from the Repo model might be the ones responsible in making the username and name downcased. :)

![screenshot from 2015-06-13 02 37 43](https://cloud.githubusercontent.com/assets/8434127/8137377/9355000e-1175-11e5-9258-98fa34529ebf.png)